Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook

```diff
@@ -100,7 +100,7 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let start = startofday({TimeRange:start});\nunion customEvents, pageViews\n| where timestamp >= start\n| where name in ({Activities}) or ('*' in ({Activities})) or ('#' in ({Activities}) and itemType == 'pageView') or ('&' in ({Activities}) and itemType == 'customEvent')\n//| summarize {Metric} by bin(timestamp, 2h)\n| make-series {Metric} default = 0 on timestamp from startofday({TimeRange:start}) to endofday({TimeRange:end}) step 2h\n| mv-expand Metric to typeof(int), timestamp to typeof(datetime)\n| extend Hour = toreal(hourofday(timestamp)) / 100, WeekDay = dayofweek(timestamp)\n| where '{ExcludeWeekends}' == 'No' or (WeekDay != 0d and WeekDay != 6d)\n| extend Day = strcat(monthofyear(timestamp), '/', dayofmonth(timestamp), ' - ', case(WeekDay == 0d, 'Sun', WeekDay == 1d, 'Mon', WeekDay == 2d, 'Tue', WeekDay == 3d, 'Wed', WeekDay == 4d, 'Thu', WeekDay == 5d, 'Fri', 'Sat'))\n| extend Day1 = bin(timestamp, 1d)\n| order by Day asc, Hour asc\n| project-away timestamp, WeekDay, Day1\n| serialize\n| evaluate pivot(Hour, sum(Metric))\n//| order by Day1 asc\n//| project-away Day1",
+        "query": "let start = startofday({TimeRange:start});\nunion customEvents, pageViews\n| where timestamp >= start\n| where name in ({Activities}) or ('*' in ({Activities})) or ('#' in ({Activities}) and itemType == 'pageView') or ('&' in ({Activities}) and itemType == 'customEvent')\n//| summarize {Metric} by bin(timestamp, 2h)\n| make-series {Metric} default = 0 on timestamp from startofday({TimeRange:start}) to endofday({TimeRange:end}) step 2h\n| mv-expand Metric to typeof(int), timestamp to typeof(datetime)\n| extend Hour = toreal(hourofday(timestamp)) / 100, WeekDay = dayofweek(timestamp)\n| where '{ExcludeWeekends}' == 'No' or (WeekDay != 0d and WeekDay != 6d)\n| extend Day = strcat(monthofyear(timestamp), '/', dayofmonth(timestamp), ' - ', case(WeekDay == 0d, 'Sun', WeekDay == 1d, 'Mon', WeekDay == 2d, 'Tue', WeekDay == 3d, 'Wed', WeekDay == 4d, 'Thu', WeekDay == 5d, 'Fri', 'Sat'))\n| extend Day1 = bin(timestamp, 1d)\n| order by localTimestamp asc, Hour asc\n| project-away timestamp, WeekDay, Day1\n| serialize\n| evaluate pivot(Hour, sum(Metric))\n//| order by Day1 asc\n//| project-away Day1",
```

Comment:
what is this localTimestamp,  you mean "timestamp"?

Reply:
Ok yeah my apologies, fixed,

Reply:
@MSKaiLian, btw that was from a clone where I'm hard coding a time diff to get local time.  Is there an existing implementation of adding the _Local_ time to Workbooks?

Example from Transaction Search:
![image](https://user-images.githubusercontent.com/50080355/173608727-b7a1d7b9-5f5d-4aa5-8852-a4bf9fe27b1e.png)

Reply:
Seem workbook time picker doesn't have the feature.

Reply:
Ok, thanks

Reply:
Are you good with resolving this conversation?